.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Hi, I recommend adding a git config that globally ignores this file, rather than doing it per-project.

There's a config item in git called `core.excludesfile` that can be used for this.

Reply:
I've been using git for years and still didn't know about this. Thanks!